### DISCUSSION TITLE
Two issues: Always need to run exec bash and "Config error" tag

### DISCUSSION THREAD

**Discussion by AmodeusR** [JanDeDobbeleer/oh-my-posh#3484] [Troubleshoot] [answered]
I tried to configured it in the bash shell, and after quite a little bit of struggle I got it working. Problem is, whenever I reopen it, the theme is not applied and I need to run exec bash every time. Besides that, there's this tag/flag or whatever how it's called saying "CONFIG ERROR". I don't have any idea of what it refers to or how to fix it. Can anyone help me with this?

![image](https://user-images.githubusercontent.com/29683600/218126224-2da9e3e5-a823-4df8-9151-36a18f22de6b.png)

**Comment by JanDeDobbeleer** [comment, marked as answer]
@AmodeusR that's Git bash, right? You need to check why it doesn't source the `.bashrc` automatically, that's beyond the scope of oh-my-posh though. As for config error, you need to use the full path to the config when using Git bash, that could be the cause. You should also see that when using `oh-my-posh debug`.

**Comment by AmodeusR** [comment]
I could fix the first issue, but the config error tag seems a little bit more puzzling to me.

With "full path to the config" you mean this?
`eval "$(oh-my-posh init bash --config ~\AppData/Local/Programs/oh-my-posh/themes/atomic.omp.json)"`
Because it's like that already. If not, what is it that needs to be the full path?

About the errors, running oh-my-posh debug shows two errors

![image](https://user-images.githubusercontent.com/29683600/218224320-ebb1a91b-b0bb-478f-8ba4-10007efe9e2b.png)
![image](https://user-images.githubusercontent.com/29683600/218224340-e22a9873-3850-4b41-b59a-0a63ac6d2026.png)

I believe it is been caused by the first error? But the path is correct till I know and the file do exist.

**Reply by lewis-yeung** [reply]
The tilde (`~`) does not work here. You should use a full path like `C:/Users/USERNAME/AppData/Local/Programs/oh-my-posh/themes/atomic.omp.json` for Git Bash. :) See the `bash` tab in [docs](https://ohmyposh.dev/docs/installation/customize#config-syntax):

> Use the full path to the config file, not the relative path or `~` as a shorthand for `$HOME`.

**Reply by AmodeusR** [reply]
Thanks, that fixed the problem!

**Reply by ghost** [reply]
How do I if it's in Ubuntu? Same config error and it's kinda hard to sort it out here because new to linux. Any hands here?

**Reply by JanDeDobbeleer** [reply]
@mshajid use `oh-my-posh debug --plain` to see what's causing the issue.

**Reply by ghost** [reply]
Thanks it worked, I didnt specified the OH MY POSH themes directory
